flutter/lib/resources/tasks.pbtxt

```diff
@@ -0,0 +1,305 @@
+task {
+  name: "Image classification"
+  dataset {
+    name: "Imagenet classification validation set"
+    type: IMAGENET
+    path: "app:///mlperf_datasets/imagenet/img"
+    groundtruth_src: "app:///mlperf_datasets/imagenet/imagenet_val_full.txt"
+  }
+  lite_dataset {
+    name: "Open images subset for classification"
+    type: IMAGENET
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/imagenet.zip"
+    groundtruth_src: ""
+  }
+  test_dataset {
+    name: "Imagenet dataset for instrument test"
+    type: IMAGENET
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/imagenet_tiny.zip"
+    groundtruth_src: "https://raw.githubusercontent.com/mlcommons/mobile_models/main/v1_0/assets/imagenet_val_tiny.txt"
+  }
+  model {
+    id: "IC_tpu_uint8"
+    name: "MobileNetEdgeTPU 224 1.0 Uint8"
+    offset: 1
+    tags: "uint8"
+    score_max: 1000
+    image_width: 224
+    image_height: 224
+  }
+  model {
+    id: "IC_tpu_float32"
+    name: "MobileNetEdgeTPU 224 1.0 F32"
+    offset: 1
+    tags: "float32"
+    score_max: 1500
+    image_width: 224
+    image_height: 224
+  }
+  min_query_count: 1024
+  min_duration_ms: 60000
+}
+
+task {
+  name: "Object detection"
+  dataset {
+    name: "COCO 2017 validation set"
+    type: COCO
+    path: "app:///mlperf_datasets/coco/img"
+    groundtruth_src: "app:///mlperf_datasets/coco/coco_val_full.pbtxt"
+  }
+  lite_dataset {
+    name: "Open images subset for detection"
+    type: COCO
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/coco.zip"
+    groundtruth_src: ""
+  }
+  test_dataset {
+    name: "Coco dataset for instrument test"
+    type: COCO
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/coco_tiny.zip"
+    groundtruth_src: "https://raw.githubusercontent.com/mlcommons/mobile_models/main/v1_0/assets/coco_val_tiny.pbtxt"
+  }
+  model {
+    id: "OD_uint8"
+    name: "SSD-MobileDET UInt8"
+    offset: 1
+    tags: "uint8"
+    score_max: 2000
+    num_classes: 91
+    image_width: 320
+    image_height: 320
+  }
+  model {
+    id: "OD_float32"
+    name: "SSD-MobileDET F32"
+    offset: 1
+    tags: "float32"
+    score_max: 5000
+    num_classes: 91
+    image_width: 320
+    image_height: 320
+  }
+  min_query_count: 1024
+  min_duration_ms: 60000
+}
+
+
+task {
```

Comment:
This task block is new compared to the original file https://raw.githubusercontent.com/mlcommons/mobile_models/main/v2_0/assets/tasks_flutterapp.pbtxt